learnyounode/04-filtered-ls.js

```diff
@@ -0,0 +1,12 @@
+var fs = require('fs');
+var fileExtension = process.argv[3];
+
+function callback (err, list) {
+  for (var i = 0; i < list.length; i++) {
+    if (list[i].includes('.' + fileExtension)) {
+      console.log(list[i]);
+    }
+  }
+};
+
+fs.readdir(process.argv[2], callback)
```

Comment:
Sure, this will work. Another useful tool for problems like this is the Node module `paths` - it has a lot of useful methods for handling and transforming file paths.